fault.c

```diff
@@ -105,12 +108,12 @@ static struct kibosh_fault_read_delay *kibosh_fault_read_delay_parse(json_value
     }
     prefix_obj = get_child(obj, "prefix");
     if ((!prefix_obj) || (prefix_obj->type != json_string)) {
-        INFO("kibosh_fault_unreadable_parse: No valid \"prefix\" field found in fault object.\n");
+        INFO("kibosh_fault_read_delay_parse: No valid \"prefix\" field found in fault object.\n");
```

Comment:
If these are error conditions, we should use ERROR instead of INFO

Reply:
I inherited the style from the current version of Kibosh for consistency. I don't think there is a ERROR macro implemented in log.h. INFO is used to output information to log files.